docs/ops.html

```diff
@@ -1841,6 +1841,63 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
       </tr>
   </tbody></table>
 
+<h4 class="anchor-heading"><a id="tiered_storage_monitoring" class="anchor-link"></a><a href="#tiered_storage_monitoring">Tiered Storage Monitoring Metrics</a></h4>
+  The following set of metrics are available for monitoring of the tiered storage feature:<br/><br/>
+  <table class="data-table">
+    <tbody>
+      <tr>
+        <th>Metric/Attribute name</th>
+        <th>Description</th>
+        <th>Mbean name</th>
```

Comment:
This looks incorrect since this says that MBean name is in third column but the it's actually in second column. 

It would help if you could please add a screenshot of how the website looks like with these changes with your next revision. You can find how to spin up website locally here: https://cwiki.apache.org/confluence/display/KAFKA/Setup+Kafka+Website+on+Local+Apache+Server 

Copy the contents of this file to kafka-site locally to see the changes.

Reply:
![Screenshot 2023-09-05 at 4 30 35 PM](https://github.com/apache/kafka/assets/138052796/954eea2a-cafb-483e-bf8d-029ba163c39c)

Can you confirm if this is what we expect for the metric names?

Reply:
<img width="1527" alt="Screenshot 2023-09-05 at 13 10 39" src="https://github.com/apache/kafka/assets/71267/34773a6a-11bd-40ed-8689-087f6dc73201">

This is how it should look right. Are you running the website locally?

Reply:
![Uploading Screenshot 2023-09-05 at 6.17.40 PM.png…]()

Reply:
![Screenshot 2023-09-05 at 6 17 40 PM](https://github.com/apache/kafka/assets/138052796/0766472c-cef8-4148-8048-1436c29bf1af)